pages/science/reactors/reactors.rst

```diff
@@ -0,0 +1,482 @@
+
+                  .. container:: card-header section-card
+
+                     Plug Flow Reactor
+                     
+                     ..
+
+
+               .. container:: card-body
+
+                  .. container:: card-text
+
+                     Derivations of governing equations for a Plug Flow Reactor
+
+In some cases, Cantera's solver is insufficient to describe 
+a certain configuration. In this situation, there are two options for customizing
+a reactor in Cantera. These two approaches are described below: Delegated Reactor and
+Custom Reactor.
+
+.. container:: container
+
+   .. row:: 
+
+      .. container:: col-12
+
+         .. container:: card-deck
+
+            .. container:: card
+
+               .. container::
+                  :tagname: a
+                  :attributes: href="delegatedreactor.html"
+                              title="Delegated Reactor"
+
+                  .. container:: card-header section-card
+
+                     Delegated Reactor
+                     
+                     ..
+
+
+               .. container:: card-body
+
+                  .. container:: card-text
+
+                     Documentation for reactor type where the user can modify existing
+                     governing equations of a chosen reactor.
+               
+            .. container:: card
+
+               .. container::
+                  :tagname: a
+                  :attributes: href="customreactor.html"
+                              title="Custom Reactor"
+
+                  .. container:: card-header section-card
+
+                     Custom Reactor
+
+               .. container:: card-body
+
+                  .. container:: card-text
+
+                     Documentation for reactor type where Cantera provides chemical and 
+                     thermodynamic computations, but external ODE solvers can be applied 
+                     to solve user supplied governing equation(s).
+               
+
+Reactor Networks
+================
+
+While reactors by themselves just define the above governing equations of the
+reactor, the time integration is performed in reactor networks. In other words
+defining a reactor without assigning it to a reactor network prevents Cantera
+from performing time integration to solve the governing equations. A reactor
+network is therefore necessary to define even if only a single reactor is considered. 
+An example of a single reactor network can be found `here
+</examples/python/reactors/combustion.html>`__.
+
+Time Integration for Reactor Networks
+-------------------------------------
+
+Cantera uses the CVODES solver from the `SUNDIALS 
+<https://computing.llnl.gov/projects/sundials>`__ 
+package to integrate the stiff ODEs of reacting systems. These stiff ODEs are referring to
+the governing equations defining the reactors above.
+More in-depth information on the CVODES solver can be found `here </science/reactors
+/cvodes.html>`__.
```

Comment:
I think we discussed this, but looking at the page it's pretty clear that this doesn't call enough attention to the fact that there's a whole additional page of info here -- I think adding a card for CVODES would make a lot of sense.